src/Interface/Application/MainWindowCollaborators.cc

```diff
@@ -310,6 +313,501 @@ QWizardPage* NewUserWizard::createOtherSettingsPage()
+  page->infoText->setText("Create a ShowField modules and turn off its ShowEdges parameter. "
+    "Connect it to the ExtractSimpleIsosurface module."
+    "\n\n"
+    "Create a ColorMap module and rescale it by connecting it to a new RescaleColorMap module, "
+    "then connecting the CalculateFieldData module to the RescaleColorMap module."
+    "\n\n"
+    "To show the color map, connect the RescaleColorMap module to the ShowField module.");
+
+  page->codeEdit->setPlainText("iso_show_field = scirun_add_module(\"ShowField\")\n"
+    "\n"
+    "scirun_set_module_state(iso_show_field, \"ShowEdges\", 0)\n"
+    "\n"
+    "scirun_connect_modules(iso, 0, iso_show_field, 0)\n"
+    "\n"
+    "color_map = scirun_add_module(\"CreateStandardColorMap\")\n"
+    "\n"
+    "rescale_color_map = scirun_add_module(\"RescaleColorMap\")\n"
+    "\n"
+    "scirun_connect_modules(color_map, 0, rescale_color_map, 0)\n"
+    "\n"
+    "scirun_connect_modules(calc, 0, rescale_color_map, 1)\n"
+    "\n"
+    "scirun_connect_modules(rescale_color_map, 0, iso_show_field, 1)");
+
+  connect(page->sendButton, &QPushButton::clicked, [this, page](){displayPython_(page->codeEdit->toPlainText());});
+  return page;
+}
+
+
+QWizardPage* PythonWizard::createViewScenePage()
+{
+  auto page = new PythonWizardCodePage;
+  page->setTitle("Working with ViewScene Module");
+  page->infoText->setText("To be able to view the volume, "
+  "you will need to create a ViewScene module and connect it to the ShowField module.");
+
+  page->codeEdit->setPlainText("view_scene = scirun_add_module(\"ViewScene\")\n"
+    "\n\n"
+    "scirun_connect_modules(iso_show_field, 0, view_scene, 0)");
+
+  connect(page->sendButton, &QPushButton::clicked, [this, page](){displayPython_(page->codeEdit->toPlainText());});
+  return page;
+}
+
+
+QWizardPage* PythonWizard::createExecutePage()
+{
+  auto page = new PythonWizardCodePage;
+  page->setTitle("Executing the Network");
+  page->infoText->setText("Execute the network by running the command 'scirun_execute_all()'");
+
+  page->codeEdit->setPlainText("scirun_execute_all()");
+
+  connect(page->sendButton, &QPushButton::clicked, [this, page](){displayPython_(page->codeEdit->toPlainText());});
+  return page;
+}
+
+QWizardPage* PythonWizard::createLoadingNetworkIntroPage()
+{
+  auto page = new QWizardPage;
+  page->setTitle("Loading a Network");
+  page->setSubTitle("This section of the wizard will demonstrate how to load then execute a network");
+  auto layout = new QVBoxLayout;
+  page -> setLayout(layout);
+  return page;
+}
+
+QWizardPage* PythonWizard::createLoadNetworkPage()
+{
+  auto page = new PythonWizardCodePage;
+  page->setTitle("Load and Execute a Network");
+  page->infoText->setText("Use the command `scirun_load_network()` to load a saved network from sample networks provided"
+    "\n\n"
+    "Execute network with `scirun_execute_all()` command");
+
+  page->codeEdit->setPlainText("scirun_load_network(\"/Users/nidhipatel/myclone/"
```

Comment:
this is hard coded to your path.  Can we load it from the scirundata path instead?  or maybe the current path?  You can get environment variables in python with `os.environ['SCIRUNDATADIR']`